### PR TITLE
Add MiroTalk to Ferdium recipes.

### PR DESCRIPTION
**Pull request by miroslavpejic85**
#### Pre-flight Checklist

Please ensure you've completed all of the following.

- [x] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-recipes/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.

#### Description of Change
Add MiroTalk to Ferdium recipes.

### CODE REVIEW

File: recipes/mirotalk-c2c/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-c2c",
+	"name": "MiroTalk C2C",
+	"version": "1.0.1",
```
Comment (vraravam) [inline recipes/mirotalk-c2c/package.json:4]: Please use `1.0.0` for the first release of a recipe.

File: recipes/mirotalk-c2c/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = Ferdium => class MiroTalkC2C extends Ferdium {
+	overrideUserAgent() {
+		return window.navigator.userAgent.replace(/(Ferdium|Electron)(\S+\s)/g, '').trim();
```
Comment (vraravam) [inline recipes/mirotalk-c2c/index.js:5]: Is this really required? If so, please specify why

File: recipes/mirotalk-c2c/package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://c2c.mirotalk.com",
+    "hasCustomUrl": true,
```
Comment (vraravam) [inline recipes/mirotalk-c2c/package.json:8]: Can this service be self-hosted? Is that why the `hasCustomUrl` is `true`? If not, please remove it

File: recipes/mirotalk-c2c/package.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+		"hasTeamId": false,
```
Comment (vraravam) [inline recipes/mirotalk-c2c/package.json:9]: If this is the default value, please remove this line

File: recipes/mirotalk-p2p/index.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = Ferdium => class MiroTalkP2P extends Ferdium {
+	overrideUserAgent() {
+		return window.navigator.userAgent.replace(/(Ferdium|Electron)(\S+\s)/g, '').trim();
```
Comment (vraravam) [inline recipes/mirotalk-p2p/index.js:5]: Same as above

File: recipes/mirotalk-p2p/package.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-p2p",
+	"name": "MiroTalk P2P",
+	"version": "1.0.1",
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://p2p.mirotalk.com",
+    "hasCustomUrl": true,
```
Comment (vraravam) [inline recipes/mirotalk-p2p/package.json:8]: same as above

File: recipes/mirotalk-p2p/package.json [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+		"hasTeamId": false,
```
Comment (vraravam) [inline recipes/mirotalk-p2p/package.json:9]: same as above

File: recipes/mirotalk-sfu/index.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = Ferdium => class MiroTalkSFU extends Ferdium {
+	overrideUserAgent() {
```
Comment (vraravam) [inline recipes/mirotalk-sfu/index.js:4]: same as above

File: recipes/mirotalk-sfu/package.json [hunk 9]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-sfu",
+	"name": "MiroTalk SFU",
+	"version": "1.0.1",
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://sfu.mirotalk.com",
+    "hasCustomUrl": true,
```
Comment (vraravam) [inline recipes/mirotalk-sfu/package.json:8]: same as above

File: recipes/mirotalk-sfu/package.json [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+		"hasTeamId": false,
```
Comment (vraravam) [inline recipes/mirotalk-sfu/package.json:9]: same as above

File: recipes/mirotalk-webrtc/index.js [hunk 11]
```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = Ferdium => class MiroTalkWEB extends Ferdium {
+	overrideUserAgent() {
```
Comment (vraravam) [inline recipes/mirotalk-webrtc/index.js:4]: same as above

File: recipes/mirotalk-webrtc/package.json [hunk 12]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-webrtc",
+	"name": "MiroTalk WebRTC",
+	"version": "1.0.1",
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://webrtc.mirotalk.com",
+    "hasCustomUrl": true,
```
Comment (vraravam) [inline recipes/mirotalk-webrtc/package.json:8]: same as above

File: recipes/mirotalk-webrtc/package.json [hunk 13]
```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-webrtc",
+	"name": "MiroTalk WebRTC",
+	"version": "1.0.1",
```
Comment (vraravam) [inline recipes/mirotalk-webrtc/package.json:4]: same as above

File: recipes/mirotalk-webrtc/package.json [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://webrtc.mirotalk.com",
+    "hasCustomUrl": true,
+		"hasTeamId": false,
```
Comment (vraravam) [inline recipes/mirotalk-webrtc/package.json:9]: same as above

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by vraravam** [comment]
There should be a few lines that are used to load `service.css` - which allows a user to override css by themselves. This is missing in all of the above. Please fix

File: recipes/mirotalk-c2c/index.js (hunk 2 above)
Comment (miroslavpejic85) [reply recipes/mirotalk-c2c/index.js:5]: Here not sorry, I'm going to fix.

File: recipes/mirotalk-c2c/package.json (hunk 3 above)
Comment (miroslavpejic85) [reply recipes/mirotalk-c2c/package.json:8]: Yes, All MiroTalk versions can be self hosted, so I want to keep "hasCustomUrl": true, to allow anyone to change the default URL with they own.

**Comment by miroslavpejic85** [comment]
> There should be a few lines that are used to load service.css - which allows a user to override css by themselves. This is missing in all of the above. Please fix


So in `webview.js` instead of 

```js
module.exports = () => {};
```

Like

```js
const _path = _interopRequireDefault(require('path'));

function _interopRequireDefault(obj) {
  return obj && obj.__esModule ? obj : { default: obj };
}

module.exports = Ferdium => {
  Ferdium.injectCSS(_path.default.join(__dirname, 'service.css'));
};
```

Thank you!

**Pull request closed without merging**